e2e/test/scenarios/data-studio/data-model/data-studio-bulk-table.cy.spec.ts

```diff
@@ -173,6 +173,81 @@ describe("bulk table operations", () => {
       .should("have.text", "Bobby Tables");
   });
 
+  describe(
+    "several databases with several schemas at once (metabase#GDGT-1275)",
+    { tags: ["@external"] },
+    () => {
+      beforeEach(() => {
+        H.restore("postgres-writable");
+        H.activateToken("bleeding-edge");
+        H.createLibrary();
+        cy.signInAsAdmin();
+        H.resetTestTable({ type: "postgres", table: "multi_schema" });
+        H.resyncDatabase({ dbId: WRITABLE_DB_ID });
+        H.DataModel.visitDataStudio();
+      });
+
+      [{ withFilter: false }, { withFilter: true }].forEach(
+        ({ withFilter }) => {
+          it(`should change metadata and see that is changed for all selected tables withFilter=${withFilter}`, () => {
+            cy.log("change the owner and check the owner column");
+            if (withFilter) {
+              TablePicker.getSearchInput().type("a");
+            } else {
+              TablePicker.getDatabase("Writable Postgres12").click();
+              TablePicker.getDatabase("Sample Database").click();
+              TablePicker.getSchema("Domestic").click();
+            }
+
+            TablePicker.getTable("Accounts")
+              .find('input[type="checkbox"]')
+              .check();
+
+            TablePicker.getTable("Animals")
+              .find('input[type="checkbox"]')
+              .check();
+            H.selectHasValue("Owner", "").click();
+            H.selectDropdown().contains("Bobby Tables").click();
+
+            ["Accounts", "Animals"].forEach((tableName) => {
+              TablePicker.getTable(tableName)
+                .findByTestId("table-owner")
+                .should("have.text", "Bobby Tables");
+            });
+
+            cy.log("publish and check publish state column");
+
+            /**
+             * we need to wait a little until sidebar is closed. Otherwise, the button will be found, but it will be immediately removed.
+             * And then we have to reselect because when there is a filter rows are unselected after a change
```

Comment:
1. Which sidebar gets closed?
2. Which button will be found but immediately removed?
3. > when there is a filter rows are unselected after a change

    Publishing tables should not remove the selection. The filter also should not change here. This may be a bug?

Reply:
1. TableAttributesEditBulk component
2. Publish button
3. TableAttributesEditBulk component is destroyed after changing attributes of selected tables because of this:
<img width="2002" height="316" alt="image" src="https://github.com/user-attachments/assets/eeb1b7f5-068b-4c64-aedb-22237789aab6" />

Reply:
Created a P3 UX bug https://linear.app/metabase/issue/GDGT-1377/the-selection-should-be-kept-after-the-changes-in-filter-mode-in-data